NCEP/docs/source/outputs.rst

```diff
@@ -2,12 +2,46 @@
 Product
 ######################
 
-The GraphCastGFS model runs 4 times a day at 00Z, 06Z, 12Z, and 18Z cycles, which applies to both 13 levels and 37-levels. The 13 pressure levels 
-are 50, 100, 150, 200, 250, 300, 400, 500, 600, 700, 850, 925, and 1000 hPa. The 37 pressure levels include 1, 2, 3, 5, 7, 10, 20, 30, 50, 70, 100, 
-125, 150, 175, 200, 225, 250, 300, 350, 400, 450, 500, 550, 600, 650, 700, 750, 800, 825, 850, 875, 900, 925, 950, 975, and 1000 hPa. 
+The GraphCastGFS model runs 4 times a day at 00Z, 06Z, 12Z, and 18Z cycles. The horizontal resolution is on 0.25 degree lat-lon grid.
+The vertical resolutions are on both 13 and 37 pressure levels.
+
+* The 13 pressure levels include:
+
+  50, 100, 150, 200, 250, 300, 400, 500, 600, 700, 850, 925, and 1000 hPa. 
+    
+* The 37 pressure levels include: 
+
+  1, 2, 3, 5, 7, 10, 20, 30, 50, 70, 100, 125, 150, 175, 200, 225, 250, 300, 350, 400, 450, 500, 550, 600, 650, 700, 750, 800, 825, 850, 875, 900, 925, 950, 975, and 1000 hPa. 
+
+The model output fields are:
+
+* 3D fields on pressure levels:
+
+  * temperature
+  
+  * U and V component of wind
+
+  * geopotential height
+
+  * specific humidity
+
+  * vertical velocity
+
+* 2D surface fields:
+
+  * 10-m U and V components of wind
+
+  * 2-m temperature
+
+  * mean sea-level pressure
+
+  *6-hourly total precipitation
+
 The near real-time forecast outputs along with inputs are available on `AWS <https://noaa-nws-graphcastgfs-pds.s3.amazonaws.com/index.html>`_. 
+
 For each cycle, the dataset contains input files to feed into GraphCast found in the directory graphcastgfs.yyyymmdd/hh/input 
```

Comment:
graphcastgfs.yyyymmdd/hh/input  in a separate line?

Reply:
@junwang-noaa Which project name will we use?
![image](https://github.com/NOAA-EMC/graphcast/assets/156125484/37a4c60e-8e98-4aff-b1aa-0b78cb6d1b3a)

Reply:
GraphCastGFS

Reply:
@LinlinCui-NOAA Please share a link of the project on `readthedocs`. Thanks

Reply:
@SadeghTabas-NOAA I'll post it here where it is ready. It takes time to build on their server.

Reply:
Sure, Thank you

Reply:
@junwang-noaa @SadeghTabas-NOAA The draft is here:
https://graphcast.readthedocs.io/en/latest/index.html

Right now, I am publishing the bugfix/docs branch of mine. After we merge this PR, I'll delete it and publish EMC/graphcast main branch.

Reply:
Great! we can test that for potential issues.

Reply:
Just to make sure, is it possible to just modify it to main branch instead of deleting? Because it might cause issues, e.g., the project name (page) is already there, and it might take time updating their server.

Reply:
Two comments: 
1) On the top left, it's said: "GraphCast UFS, can we change it to "GraphCast GFS"
2) On the right panel, can we change "graphcast Global Forecast System" to "GraphCast with GFS intput"

Reply:
I can import another project. And I did not change project name for this one, as you can see the url is `graphcast` instead of `graphcastgfs`.

Reply:
@LinlinCui-NOAA It also would be good to provide a link to the EMC/graphcast main branch for the source code

Reply:
@SadeghTabas-NOAA When you click on `Edit on GitHub`, it will take you to the main branch if it is published from there.

Reply:
you can put it in the installation page

Reply:
@junwang-noaa 
> On the top left, it's said: "GraphCast UFS, can we change it to "GraphCast GFS"

Good catch.

> On the right panel, can we change "graphcast Global Forecast System" to "GraphCast with GFS intput"

Do you mean the index page using "GraphCast with GFS input"

Reply:
> @SadeghTabas-NOAA When you click on `Edit on GitHub`, it will take you to the main branch if it is published from there.

I know about this, but that is a functionality to edit the doc. What I mean is to add a few sentence on where to clone the source code before running it.

Reply:
@SadeghTabas-NOAA Just added.

Reply:
no other comment from my side.